ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java

```diff
@@ -22,7 +22,7 @@
     private static String REST_PATH_2 = "'\"]";
     private static By CALENDAR_EYE_FORM = By.id("w1-display");
     private static By DATAPICKER_YEAR = By.className("ui-datepicker-year");
-    private static By DATAPICKER_MONTH = By.className("ui-datepicker-moth");
+    private static By DATAPICKER_MONTH = By.className("ui-datepicker-month");
     private static By DATAPICKER_DAY = By.className("ui-state-default");
```

Comment:
could you link the page with this className on github to be sure this was a type

Reply:
This is no where to be found on the legacy ui, neither under manageForms especially on manageFormsPage, According to this functionality,By the time of this functionality , manageForms used to capture date, but later this functionality was removed, so ideally we are not seeing where All Date functions being used

Reply:
I guess DatePicker functions should be removed here

Reply:
is this page setting the month or using `DATAPICKER_MONTH` element? if it does what's the current existing element class/reference?

Reply:
Am closing this ticket for now , since it doesnot have this element, this element is nolonger being used on managefFormsPage, i have checked on different manageFormsPage in legacy ui, referenceapplication , but couldnt find DatePickers, However DatePickers(MONTH,YEAR) are nolonger being used on this page